src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/Contract.Message.Tests.csproj

```diff
@@ -17,9 +17,10 @@
     <ProjectGuid>{A835C2BC-DFD0-4ED6-A240-59FBEE7F0157}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="MessageContractCommon.cs" />
+    <Compile Include="MessageContractCommon.4.0.0.cs" />
+    <Compile Include="MessageContractTests.4.0.0.cs" />
+    <Compile Include="MessageTests.4.0.0.cs" />
     <Compile Include="MessageContractTests.cs" />
```

Comment:
Another place where "**\*.cs* works better.  I'll stop commenting on this suggestion.